### DISCUSSION TITLE
Bi-Weekly Community Meeting, July 13, 2024

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3541] [Announcements]
Everyone is invited to the next TaskWarrior community bi-weekly meeting!  
Want to get a sense of what to expect? Watch the recent meeting videos here…

- [June 19](https://www.youtube.com/watch?v=M8BRQJfRgoE)
- [June 15](https://www.youtube.com/watch?v=qI9P1L8fg0s)
- [June 1](https://youtu.be/hZQG8jDn24I)

Our next meeting is 12:00 UTC Saturday, July 13, 2024 ([check that in your time zone @ timeanddate.com](https://www.timeanddate.com/worldclock/meetingtime.html?month=7&day=13&year=2024&p1=5&p2=3904&p3=3918&p4=3921&iv=0)). You can [add this event to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MW45bHUzc3VuazNicnI2YmVkNXBydmNmOHUgd2VzbGV5QG13cGVubmVyLmNvbQ&tmsrc=wesley%40mwpenner.com).

Google Meet joining info
Video call link: https://meet.google.com/tns-suqq-iaj
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

The formal agenda portion of the meeting runs ~30 minutes. The room will stay open for anyone who wants to hang out or go deeper into any of the topics from the agenda.

**Comment by wrashi** [comment]
Tomorrow we'll be doing our bi-weekly community meeting, as noted above. Just want to extend the invitation to join for anyone sitting on the fence…

**Reply by felixschurk** [reply]
I am still on the road so will not be able to make it today again :(

**Reply by wrashi** [reply]
Sorry to hear that, @felixschurk! Wishing you safe travels.

**Comment by wrashi** [comment]
The room is now open for anyone who wants to join.
https://meet.google.com/tns-suqq-iaj

**Reply by wrashi** [reply]
I'm excited to post the video from our community meeting the same day as the meeting! Check it out…

https://youtu.be/cJE7RBZuWCU

**Comment by djmitche** [comment]
I'll be in Vancouver, BC for IETF during the next meeting. I think 5am is a little early for me, so I won't make it :)

If any other TW folks are at IETF, I'd love to say hi!

At the meeting after that, maybe we can do a demo of how the cloud sync works, looking at the contents of the bucket in the cloud console. It's just complicated enough to be interesting while still understandable, and it would be good to share understanding of how sync data is stored and its privacy characteristics.

**Reply by wrashi** [reply]
What if instead of July 27, we postponed the meeting till August 3? 

Also, FWIW I would be extremely interested in the cloud sync demo!

**Reply by djmitche** [reply]
I'd like to not be "special" in the project, meaning the meeting could go on without me. But I suppose that's not the reality of the situation right now. What do others think?

**Reply by wrashi** [reply]
Hmm… I didn't think of it as you being special so much as bi-weekly being flexible to fit the needs of those who attend. Bi-weekly seems to be a good cadence, but it would be normal if the project speed, especially in summer, flagged a bit.

**Reply by djmitche** [reply]
I could do the 3rd! It's up to you, really.

**Reply by wrashi** [reply]
Since there hasn't been any strong opinions expressed one way or the other beyond ours, why don't we aim for the 3rd and chalk it up to "summer scheduling".

**Reply by djmitche** [reply]
Sounds good!
